help/destinations/catalog/email-marketing/salesforce-pardot.md

```diff
@@ -0,0 +1,203 @@
+
+Note down the items below before you authenticate to the [!DNL Salesforce Pardot] destination.
+
+| Credential | Description |
+| --- | --- |
+| `Username` | Your [!DNL Salesforce Pardot] account username. |
+| `Password` | Your [!DNL Salesforce Pardot] account password. |
+| `Pardot Business Unit Id` | To find the Account Engagement Business Unit ID, use Setup in [!DNL Salesforce]. From Setup, enter *Business Unit Setup* in the Quick Find box. Your Account Engagement Business Unit ID begins with `0Uv` and is 18 characters long. If you can’t access the Business Unit Setup information, ask your [!DNL Salesforce] Administrator to provide you with the `Account Engagement Business Unit ID`. If you require any additional guidance refer to the [[!DNL Salesforce] Authentication](https://developer.salesforce.com/docs/marketing/pardot/guide/authentication) guideline page. |
+| `Client ID` | The `Consumer Key` for your [[!DNL Salesforce] connected app](#prerequisites-connected-app). <br> Refer to the [[!DNL Salesforce] documentation](https://help.salesforce.com/s/articleView?id=sf.connected_app_rotate_consumer_details.htm&type=5) to learn how to obtain this value from the [!DNL Salesforce] interface. |
+| `Client Secret` | The `Consumer Secret` for your [[!DNL Salesforce] connected app](#prerequisites-connected-app). <br> Refer to the [[!DNL Salesforce] documentation](https://help.salesforce.com/s/articleView?id=sf.connected_app_rotate_consumer_details.htm&type=5) to learn how to obtain this value from the [!DNL Salesforce] interface. |
+
+{style="table-layout:auto"}
+
+### Guardrails {#guardrails}
+
+* Refer to the [!DNL Salesforce Pardot] [rate limits](https://developer.salesforce.com/docs/marketing/pardot/guide/overview.html#rate-limits) which details the limits imposed by your plan and would also apply to the Experience Platform executions.
+
+* Your [!DNL Salesforce Pardot] account and [!DNL Salesforce] connected app should be setup with any permissions mentioned in [You need to have a [!DNL Salesforce Pardot] account](#prerequisites-account) and [Configure a connected app within [!DNL Salesforce]](#prerequisites-connected-app) sections.
+
+## Supported identities {#supported-identities}
+
+[!DNL Salesforce Pardot] supports the activation of identities described in the table below. Learn more about [identities](/help/identity-service/namespaces.md).
+
+| Target Identity | Description | Considerations |
+|---|---|---|
+| Email | Prospect Email Address | Mandatory |
+
+## Export type and frequency {#export-type-frequency}
+
+Refer to the table below for information about the destination export type and frequency.
+
+| Item | Type | Notes |
+---------|----------|---------|
+| Export frequency | **[!UICONTROL Batch]** | Batch destinations export files to downstream platforms in increments of three, six, eight, twelve, or twenty-four hours. Read more about [batch file-based destinations](/help/destinations/destination-types.md#file-based).|
```

Comment:
I don't think this is correct. This destination does not export files to downstream platforms. This is an API-based integration.
I think we should reuse the same information in this doc as in the same section from the (API) Salesforce Marketing Cloud destination:

https://experienceleague.adobe.com/docs/experience-platform/destinations/catalog/email-marketing/salesforce-marketing-cloud-exact-target.html?lang=en#export-type-frequency